tigerpath/majors_and_certificates/majors/GER.json

```diff
@@ -16,8 +16,8 @@
     },
     {
       "type": "Director of Undergraduate Studies",
-      "name": "Thomas Levin",
-      "email": "tylevin@princeton.edu"
+      "name": "Johannes Wankhammer",

```

Comment:
https://ua.princeton.edu/academic-units/department-german

Reply:
Looks like there are two directors now, with Thomas Levin being one of them. I'm inclined to just leave it as is since Thomas Levin's name appears first anyway.

Reply:
Or actually, maybe you could add another contact with Johannes Wankhammer's name. So there'll be two entries for the Director of Undergraduate Studies.

Reply:
Updated